doc/release/upcoming_changes/23762.new_function.rst

```diff
@@ -1,5 +1,5 @@
 Matrix transpose support for ndarrays
 -------------------------------------
 NumPy now offers support for calculating the matrix transpose of an array. The
-matrix transpose is equivalent to swapping the last two axes of an array.
-`np.ndarray` and `np.ma.MaskedArray` now exposes a `.mT` attribute.
\ No newline at end of file
+matrix transpose is equivalent to swapping the last two axes of an array. The
```

Comment:
I'd leave out the new "the" and say "Both `np.narray` and ...".  ".mT" and the rest should be in double backticks.

Reply:
Fixed in new commit